doc/tutorials/rtvm_workload_design_guideline.rst

```diff
@@ -0,0 +1,46 @@
+.. _rtvm_workload_guideline:
+
+Guidelines to the workload design in an RTVM
+############################################
+
+An RTOS developer must be aware of the differences between running applications on a native
+platform and a real time virtual machine (RTVM), especially issues impacting real time
+performance. For example, a real time thread should avoid triggering any VM-Exit. If a VM-Exit
+is triggered, the developer must account for an additional margin of CPU cycles for the
+incremental runtime.
+
+This document provides some workload design guidelines when using an RTVM within the ACRN hypervisor.
+
+Run RTVM with dedicated resources/devices
+*****************************************
+
+For best practice, ACRN allocates dedicated CPU, memory resources, and cache resources (using Intel
+Cache Allocations Technology, aka CAT) to RTVMs. For best real time performance of I/O devices,
+we recommend using dedicated (pass-thru) PCIe devices to avoid VM-Exit at run time.
+
+.. note::
+   The configuration space for pass-thru PCI devices is still emulated and the access will
+   trigger a VM-Exit.
+
+RTVM with virtio PMD (Polling Mode Driver) for I/O sharing
+**********************************************************
+
+If the RTVM must use shared devices, we recommend using PMD drivers that can eliminate the
+unpredictable latency caused by guest I/O trap-and-emulate access. The RTVM workload must be
+aware that the packets in the PMD driver may arrive or be sent later than expected. The workload
+in RTVM must be aware of that for better RT behavior.
```

Comment:
We can delete this last sentence: ``The workload in ... for better RT behavior.``